src/main/java/net/eldeen/dropwizard/CfSignalResourceConfig.java

```diff
@@ -0,0 +1,48 @@
+package net.eldeen.dropwizard;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.Configuration;
+import org.hibernate.validator.constraints.NotBlank;
+
+/**
+ *
+ */
+public class CfSignalResourceConfig extends Configuration {
+
+  @NotBlank
+  private String asgResourceName;
```

Comment:
Is ASG the only type of resource that could be signaled? If not this property should be more generically named.